docs/daemon/network.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 4
+sidebar_label: "Testnet"
```

Comment:
"Testnet" is seeming stranger since we talked.  We don't have a testnet, so I'd bet this will confuse everyone thinking this is where to look for information on testnet, which doesn't exists.  Can the menu item also be "Network Setup" (like the title) or maybe "Network"?

Reply:
Yeah, removed it